pygeo/mphys/mphys_dvgeo.py

```diff
@@ -84,23 +130,29 @@ def compute(self, inputs, outputs):
         # next time the jacvec product routine is called
         self.update_jac = True
 
-    def nom_addChild(self, ffd_file):
+    def nom_addChild(self, ffd_file, DVGeoName=None):
+        # if we have multiple DVGeos use the one specified by name
+        if self.multDVGeo:
+            DVGeo = self.DVGeos[DVGeoName]
+        else:
+            DVGeo = self.DVGeo
+
         # can only add a child to a FFD DVGeo
-        if self.geo_type != "ffd":
+        if DVGeo.geoType != "ffd":
```

Comment:
One suggestion: Instead of adding `geoType` to each parameterization class, use `isinstance` to check what class `DVGeo` is. For example, this line would be `if not isinstance(DVGeo, DVGeometry):`. This is less error prone than using strings.

Reply:
Good point, I updated these to all directly check `type`